### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by josebarata**
[#131350487] Add footer

### CODE REVIEW

File: service/templates/base.html [hunk 1]
```diff
@@ -71,7 +71,7 @@
                     <span>Skip to main content</span>
                 </a>
                 <a class="logo" href="//jisc.ac.uk">
-                    <img alt="Jisc logo" src="{{url_for('static', filename='images/jisc-logo.png')}}">
+                     <img alt="Jisc logo" width="43" height="25" src="{{url_for('static', filename='images/jisc-logo.png')}}">
```
Comment (tharglet) [inline service/templates/base.html:74]: Is this the correct way of doing said image? Should original be resized, or do sizes belong here or in CSS?
@TomYeahOK - thoughts?

Comment (josebarata) [reply service/templates/base.html:74]: Just a note: The same logo image is used in 2 places - footer and banner. It has to be resized at least for the banner. The previous image was w43xh25 (used by CL) the current one is w316xh184.

Comment (tdcodesthings) [reply service/templates/base.html:74]: Makes sense to use just one image to cut down requests, and scale depending on where it needs to sit. Does it make sense to use an SVG (for better scaling and smaller file sizes) (or does that complicate matters)?

@tharglet Re: where to define width, colleagues are telling me CSS (whether png or css) is better.

Comment (tharglet) [reply service/templates/base.html:74]: Having a quick look around the internet, IE11 might still be having issues with svg (http://caniuse.com/#feat=svg), so I think it's probably best to leave the logo as a png for now.

**Pull request closed without merging**